src/pages/gateway/headers.md

```diff
@@ -124,6 +124,65 @@ Mesh owners can use the `responseConfig.headers` object to add response headers.
     }
 ```
 
+#### Forward headers
+
+The `responseConfig.headers` object also allows you to return header values from a source. In the following example, we are requesting the `X-Magento-Cache-Id` and `X-Cache` headers from the Venia source.
+
+```json
+{
+  "meshConfig": {
+    "sources": [
+      {
+        "name": "venia",
+        "handler": {
+          "graphql": {
+            "endpoint": "https://venia.magento.com/graphql"
+          }
+        },
+        "responseConfig": {
+          "headers": [
+            "X-Magento-Cache-Id",
+            "X-Cache"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+##### Preventing conflicting return header values
+
+When forwarding headers, an error occurs if two or more sources return a header with the same name. To prevent this, include the `x-include-metadata` header with a value of `true` in the request.
```

Comment:
There isn't a causal relation between duplicate headers and the x-include-metadata header. If a mesh creator has duplicate values in two different sources, they will get an error whether or not they have included the x-include-metadata header. 

Perhaps replace the phrase "To prevent this" to something like "To see all headers, regardless of whether they are included in the responseConfig"